service/hub/utils/map.go

```diff
@@ -0,0 +1,46 @@
+package utils
+
+import (
+	"sync"
+	"time"
+)
+
+type MapWithReset struct {
```

Comment:
This map structure seems to be designed for counting, so it should be named with related functions, like CountMap。

Reply:
refactor to limiter, this map is deprecated.